models/finances/index.ts

```diff
@@ -0,0 +1,19 @@
+import { clientFinance, IFinance } from '#clients/finance';
+import { getUniteLegaleFromSlug } from '#models/unite-legale';
+import { IUniteLegale } from '..';
+
+export interface IFinancesFromSlug {
+  uniteLegale: IUniteLegale;
+  finances: IFinance[];
+}
+
+export const getFinancesFromSlug = async (
+  slug: string
+): Promise<IFinancesFromSlug> => {
```

Comment:
we should : 
- add a verify siren thena  promise all
- add a try catch to handle httpNotFound and other exceptions from `clientFinance` (for instance https://test.annuaire-entreprises.data.gouv.fr/finances/839517323)

Reply:
`models/espace-agent/subventions-associations`does pretty much the same